Nodejs/Product/AssemblyVersion.cs

```diff
@@ -1,64 +0,0 @@
-//*********************************************************//
-//    Copyright (c) Microsoft. All rights reserved.
-//
-//    Apache 2.0 License
-//
-//    You may obtain a copy of the License at
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-//    Unless required by applicable law or agreed to in writing, software
-//    distributed under the License is distributed on an "AS IS" BASIS,
-//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
-//    implied. See the License for the specific language governing
-//    permissions and limitations under the License.
-//
-//*********************************************************//
-
-using System.Reflection;
-using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
-
-// If you get compiler errors CS0579, "Duplicate '<attributename>' attribute", check your
-// Properties\AssemblyInfo.cs file and remove any lines duplicating the ones below.
-// (See also AssemblyInfoCommon.cs in this same directory.)
-
-#if !SUPPRESS_COMMON_ASSEMBLY_VERSION
-[assembly: AssemblyVersion(AssemblyVersionInfo.StableVersion)]
-#endif
-[assembly: AssemblyFileVersion(AssemblyVersionInfo.Version)]
-
-class AssemblyVersionInfo {
-    // This version string (and the comment for StableVersion) should be
-    // updated manually between major releases (e.g. from 1.0 to 2.0).
-    // Servicing branches and minor releases should retain the value.
-    public const string ReleaseVersion = "1.0";
-
-    // This version string (and the comment for Version) should be updated
-    // manually between minor releases (e.g. from 1.0 to 1.1).
-    // Servicing branches and prereleases should retain the value.
-    public const string FileVersion = "1.1";
-
-    // This version should never change from "4100.00"; BuildRelease.ps1
-    // will replace it with a generated value.
-    public const string BuildNumber = "4100.00";
```

Comment:
Deleting the entire file seems like overkill - all I was saying is not to change the above version string :smiley: